components/Nav.js

```diff
@@ -18,6 +18,11 @@ const links = {
   Leaderboard: {
     href: '/leaderboard',
   },
+  Stories, Art, Toons: {
+    href: '/blog/category/138',
+    as: '/blog/category/138',
```

Comment:
What is this "as" here for? Why is it needed?

Reply:
On an afterthought ... it can be removed indeed.

Reply:
This would be how it could look like. 

// Constants
const links = {
  News: {
    href: '/blog/all',
    as: '/blog',
  },
  Statistics: {
    href: '/statistics',
  },
  Leaderboard: {
    href: '/leaderboard',
  },
  Stories_Art_Toons: {
    href: '/blog/category/138',
  },
  
}

Reply:
Object property names can be declared as strings for complex names. ```Stories_Art_Toons``` can be ```'Stories, Art, & Toons'```